eras/shelley/impl/src/Cardano/Ledger/Shelley/LedgerState.hs

```diff
@@ -536,11 +543,53 @@ pvCanFollow _ SNothing = True
 pvCanFollow (ProtVer m n) (SJust (ProtVer m' n')) =
   (m + 1, 0) == (m', n') || (m, n + 1) == (m', n')
 
+-- =============================
+
+-- | Incremental Stake, Stake along with possible missed coins from danging Ptrs
```

Comment:
Can you elaborate the scenario described in this comment?

Reply:
It now reads like this.

#-- | Incremental Stake, Stake along with possible missed coins from danging Ptrs.
--   Transactions can use Ptrs to refer to a stake credential in a TxOut. The Ptr
--   does not have to point to anything until the epoch boundary, when we compute
--   rewards and aggregate staking information for ranking. This is unusual but legal.
--   In a non incremental system, we use what ever 'legal' Ptrs exist at the epoch
--   boundary. Here we are computng things incrementally, so we need to remember Ptrs
--   that might point to something by the time the epoch boundary is reached. When
--   the epoch boundary is reached we 'resolve' these pointers, to see if any have
--   become non-dangling since the time they came into the incremental computation.